### PR TITLE
poll db

### LINKED ISSUE #5: Server(time permitting): Provide an API for client polling of new results

### PR DESCRIPTION
**Pull request by tbushman**
Closes #5

### CODE REVIEW

**Comment by Bouncey** [comment]
LGTM!!

Pulling down for a QA

**Comment by Bouncey** [comment]
I'm not getting anything for the leaderboard

`team.lighthouse is undefined`

**Comment by tbushman** [comment]
I am unable to test locally at the moment, but it should be self.team.lighthouse when in the vue component

**Comment by tbushman** [comment]
OK, I see what you mean now. The interval polls the whole database, not just one team.

**Comment by tbushman** [comment]
Sorry, the error is in the lighthouse sorting method. I will create a PR to fix that.

File: server/views/layout.pug [hunk 1]
```diff
@@ -33,20 +33,26 @@ html(lang='en')
...
-          }
+            lighthousekeys: [ 'seo', 'best-practices', 'accessibility', 'pwa', 'performance'],
+            teamsInterval: ''
+          } 
         },
         mounted: function() {
           var self = this;
           self.log();
           if (self.teams && self.teams !== '' && self.teams !== ['']) {
             self.getTeams();
```
Comment (tbushman) [inline server/views/layout.pug:44]: This was key. In a previous commit, I had removed it and replaced it with startTeamsPolling() below it.

**Review by Bouncey — APPROVED** [review]

**Pull request merged**